Makefile

```diff
@@ -129,12 +129,18 @@ test-python-ci:
 		--ignore tests/sentry/eventstream/kafka \
 		--ignore tests/sentry/snuba \
 		--ignore tests/sentry/search/events \
+		--ignore tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py \
 		--cov . --cov-report="xml:.artifacts/python.coverage.xml" --junit-xml=".artifacts/python.junit.xml" || exit 1
 	@echo ""
 
 test-snuba:
 	@echo "--> Running snuba tests"
-	pytest tests/snuba tests/sentry/eventstream/kafka tests/sentry/snuba tests/sentry/search/events -vv --cov . --cov-report="xml:.artifacts/snuba.coverage.xml" --junit-xml=".artifacts/snuba.junit.xml"
+	pytest tests/snuba \
+		tests/sentry/eventstream/kafka \
+		tests/sentry/snuba \
+		tests/sentry/search/events \
+		tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py \
```

Comment:
There's also `test-relay-integration` which can run Kafka etc, could we rather add it there? Relay and the ingest consumers are closer to each other -- in fact, Relay integration tests also partially cover ingest consumers already.

Reply:
Oh whoops, I forgot I set this to automerge. that's a better place for it, yeah